README.md

```diff
@@ -33,8 +33,8 @@ Create a config file somewhere, for example `./config/local-config.edn`:
  :spotify/client-secret "3abdc"}
 ```
 
-Since we want to check the config into source control, but we don't want to
-check in our client-secret, we'll encrypt it.
+We want to check the prod config into source control, but we don't want to
```

Comment:
The "since" is so far away from the end that the sentence was a little heavy reading.